includes/Rocket.h

```diff
@@ -56,6 +57,10 @@ class Rocket {
 
     Vector3d get_Cp_vect() const { return Cp_vect_; };
 
+    Vector3d get_r_geod() const { return r_geod_; };
```

Comment:
We shouldn't have two separate variables holding coordinates that should always be equal. This function should use the coordinate transformation functions to get the geod coordinates from r_vect